Makefile.am

```diff
@@ -60,10 +60,6 @@ V_COPY = $(V_COPY_$(V))
 V_COPY_ = $(V_COPY_$(AM_DEFAULT_VERBOSITY))
 V_COPY_0 = @echo "  COPY    " $@;
 
-V_TAR = $(V_TAR_$(V))
```

Comment:
Dropping this here looks wrong -- it's still being used by `install-data-local` and `dist-hook`.

Reply:
`Makefile.am` includes `pkg/build` and uses many things that are defined there, including the `V_COPY`, just above as well..